clog/clog.go

```diff
@@ -105,6 +105,16 @@ func ToBufferAtLevel(lvl string) *bytes.Buffer {
 	return buff
 }
 
+// ContextWithFileHandler returns a context with a log handler at the given level.
```

Comment:
returns a context that will log to the given file at the given level.

Reply:
Updated